From 30b6f03209e0005756b6f8c9e135d75b28e67028 Mon Sep 17 00:00:00 2001
From: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>
Date: Wed, 24 Sep 2025 07:02:26 +0000
Subject: [PATCH] Revert kernelflinger patch

revert commit id# 38b71582d0d93b11e4710f5c75f2b1d5cafa2261

Signed-off-by: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>

diff --git a/libkernelflinger/android.c b/libkernelflinger/android.c
index 9944529..58ea694 100644
--- a/libkernelflinger/android.c
+++ b/libkernelflinger/android.c
@@ -479,6 +479,12 @@ static inline EFI_STATUS handover_jump(EFI_HANDLE image,
         return ret;
 
 boot:
+#ifdef USE_TRUSTY
+        /*
+         * Called after ExitBootService.
+         */
+        trusty_late_init();
+#endif
 
 #if __LP64__
         /* The 64-bit kernel entry is 512 bytes after the start. */
-- 
2.34.1

