From f099cbc3cf8af3a431058b1c9417974ef99f989d Mon Sep 17 00:00:00 2001
From: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>
Date: Mon, 30 May 2022 14:54:27 +0530
Subject: [PATCH 01/18] Enable Remote Inferencing in CIV

Setup host environment for remote inferencing

Tracked-On: OAM-102337
Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>
Signed-off-by: nitishat <nitisha.tomar@intel.com>
Signed-off-by: akodanka <anoob.anto.kodankandath@intel.com>

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index ec62bdf..48330df 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -221,6 +221,11 @@ function ubu_update_fw(){
     reboot_required=1
 }
 
+function setup_remote_infer() {
+    $CIV_WORK_DIR/scripts/setup_rinfer_host.sh
+    reboot_required=1
+}
+
 function check_os() {
     local version=`cat /proc/version`
 
@@ -468,6 +473,7 @@ function show_help() {
     printf "\t-h  show this help message\n"
     printf "\t-u  specify Host OS's UI, support \"headless\" and \"GUI\" eg. \"-u headless\" or \"-u GUI\"\n"
     printf "\t--auto-start auto start CiV guest when Host boot up.\n"
+    printf "\t-i  enable remote inferencing\n"
 }
 
 function parse_arg() {
@@ -497,6 +503,10 @@ function parse_arg() {
                 start_thermal_daemon || return -1
                 ;;
 
+            -i)
+                setup_remote_infer || return -1
+                ;;
+
             --auto-start)
                 install_auto_start_service "$2" || return -1
                 shift
-- 
2.40.0

