From 4a96c509b3838cc40a82ead0dee83b88d28c3341 Mon Sep 17 00:00:00 2001
From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
Date: Thu, 1 Jun 2023 19:31:04 +0530
Subject: [PATCH] Add parameter to disable host input.

Tracked-On:
Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>
---
 sample/guest01.ini | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sample/guest01.ini b/sample/guest01.ini
index 225402a..d2c687d 100644
--- a/sample/guest01.ini
+++ b/sample/guest01.ini
@@ -77,6 +77,7 @@ audio_type=legacy-hda
 #connectors_0=
 #connectors_1=
 #show_fps=
+#disable_host_input=no
 
 [passthrough]
 #specified the PCI id here if you want to passthrough it to guest, seperate them with comma
-- 
2.40.1

