From f9a72fc2651a6772333908e3206f0ba541e5840b Mon Sep 17 00:00:00 2001
From: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>
Date: Thu, 28 Aug 2025 13:19:55 +0000
Subject: [PATCH] Define ANDROID config in Kconfig

Change-Id: Ic9ebdb537b3243b21397256c27b75e2430d3f9e
Signed-off-by: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>

diff --git a/drivers/android/Kconfig b/drivers/android/Kconfig
index 4f8b32fc248f..963057bff685 100644
--- a/drivers/android/Kconfig
+++ b/drivers/android/Kconfig
@@ -1,6 +1,13 @@
 # SPDX-License-Identifier: GPL-2.0
 menu "Android"
 
+config ANDROID
+        bool "Android Drivers"
+        help
+          Enable support for various drivers needed on the Android platform
+
+if ANDROID
+
 config ANDROID_BINDER_IPC
 	bool "Android Binder IPC Driver"
 	depends on MMU
@@ -123,4 +130,6 @@ config ANDROID_VENDOR_OEM_DATA
 
 	  If even slightly unsure, say Y.
 
+endif # if ANDROID
+
 endmenu
diff --git a/drivers/staging/android/Kconfig b/drivers/staging/android/Kconfig
index 4d14229849ca..8e7992fac5d4 100644
--- a/drivers/staging/android/Kconfig
+++ b/drivers/staging/android/Kconfig
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: GPL-2.0
 menu "Android"
 
+if ANDROID
+
 config ANDROID_STAGING
 	bool
 
@@ -38,4 +40,6 @@ config ANDROID_FWDATA
 	  the required information can be found in sysfs with the expected
 	  layout.
 
+endif # if ANDROID
+
 endmenu
-- 
2.34.1

