From a37cfd02252f66ede4a23127acd7e05ada83ea8c Mon Sep 17 00:00:00 2001
From: "Chenthati, Pradeep" <pradeepx.chenthati@intel.com>
Date: Thu, 25 May 2023 22:59:03 +0530
Subject: [PATCH] WA add raptorlake platform for disabling pmu

Tracked-On: OAM-110419
Signed-off-by: Chenthati, Pradeep <pradeepx.chenthati@intel.com>
---
 arch/x86/events/core.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/events/core.c b/arch/x86/events/core.c
index 15ff4e3b639b6..d11d5993a79c5 100644
--- a/arch/x86/events/core.c
+++ b/arch/x86/events/core.c
@@ -251,6 +251,8 @@ bool check_hw_exists(struct pmu *pmu, int num_counters, int num_counters_fixed)
 
 	if(boot_cpu_has(X86_FEATURE_HYPERVISOR) &&
 			(boot_cpu_data.x86_model == INTEL_FAM6_ALDERLAKE ||
+			 boot_cpu_data.x86_model == INTEL_FAM6_RAPTORLAKE ||
+			 boot_cpu_data.x86_model == INTEL_FAM6_RAPTORLAKE_P ||
 			boot_cpu_data.x86_model == INTEL_FAM6_ALDERLAKE_L)) {
 		pr_cont("Disabling PMU for VM on ADL.\n");
 		goto msr_fail;
-- 
2.40.1

