From d90956b1354be09f158aec84396b0944ceca5446 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Tue, 4 Jul 2023 18:06:02 +0530
Subject: [PATCH] Clang fix for libva

Tracked-On: OAM-111115
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

diff --git a/va/va.c b/va/va.c
index 71a5981..5cd7f3f 100644
--- a/va/va.c
+++ b/va/va.c
@@ -381,8 +381,10 @@ static VAStatus va_getDriverNameByIndex(VADisplay dpy, char **driver_name, int c
     driver_name_env = getenv("LIBVA_DRIVER_NAME");
     /*if user set driver name by vaSetDriverName */
     if (ctx->override_driver_name) {
-        if (*driver_name)
+        if (*driver_name!=NULL){
             free(*driver_name);
+	    *driver_name = NULL;
+	}
         *driver_name = strdup(ctx->override_driver_name);
         if (!(*driver_name)) {
             va_errorMessage(dpy, "va_getDriverNameByIndex  failed with %s, out of memory\n", vaErrorStr(VA_STATUS_ERROR_ALLOCATION_FAILED));
@@ -391,8 +393,10 @@ static VAStatus va_getDriverNameByIndex(VADisplay dpy, char **driver_name, int c
         va_infoMessage(dpy, "User requested driver '%s'\n", *driver_name);
         return VA_STATUS_SUCCESS;
     } else if (driver_name_env && (geteuid() == getuid())) {
-        if (*driver_name)
+        if (*driver_name!=NULL){
             free(*driver_name);
+	    *driver_name = NULL;
+	}
         /*if user set driver name by environment variable*/
         *driver_name = strdup(driver_name_env);
         va_infoMessage(dpy, "User environment variable requested driver '%s'\n", *driver_name);
@@ -736,8 +740,10 @@ VAStatus vaInitialize(
     }
     /*load driver one by one, until load success */
     for (candidate_index = 0; candidate_index < num_candidates; candidate_index ++) {
-        if (driver_name)
+        if (driver_name != NULL){
             free(driver_name);
+	    driver_name = NULL;
+	}
         vaStatus = va_getDriverNameByIndex(dpy, &driver_name, candidate_index);
         if (vaStatus != VA_STATUS_SUCCESS) {
             va_errorMessage(dpy, "vaGetDriverNameByIndex() failed with %s, driver_name = %s\n", vaErrorStr(vaStatus), driver_name);
@@ -755,8 +761,10 @@ VAStatus vaInitialize(
     *major_version = VA_MAJOR_VERSION;
     *minor_version = VA_MINOR_VERSION;
 
-    if (driver_name)
+    if (driver_name != NULL){
         free(driver_name);
+        driver_name = NULL;
+    }
 
     VA_TRACE_LOG(va_TraceInitialize, dpy, major_version, minor_version);
     VA_TRACE_RET(dpy, vaStatus);
-- 
2.41.0

