From 6c800ddc828ff1b8647f32248eb63db1d0654adb Mon Sep 17 00:00:00 2001
From: celadon <celadon@intel.com>
Date: Thu, 3 Jul 2025 05:16:39 +0000
Subject: [PATCH] include lts2024 kernel

Signed-off-by: celadon <celadon@intel.com>

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index f9e4e876..7453912d 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -48,6 +48,10 @@ else ifeq ($(BASE_LINUX_INTEL_LTS2023_KERNEL), true)
 else ifeq ($(BASE_LTS2024_ANDROID_KERNEL), true)
   LOCAL_KERNEL_SRC := {{{lts2024_android_src_path}}}
   KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/{{{lts2024_android_cfg_path}}}
+else ifeq ($(BASE_LINUX_INTEL_LTS2024_KERNEL), true)
+  LOCAL_KERNEL_SRC := {{{linux_intel_lts2024_src_path}}}
+  KERNEL_CONFIG_PATH := $(TARGET_DEVICE_DIR)/{{{linux_intel_lts2024_cfg_path}}}
+  ENABLE_I915_OOT_MODULE_LOADING := true
 else
   LOCAL_KERNEL_SRC := {{{src_path}}}
   EXT_MODULES := {{{external_modules}}}
diff --git a/groups/kernel/BoardConfig.mk b/groups/kernel/BoardConfig.mk
index ffa80ea4..05aeba2a 100644
--- a/groups/kernel/BoardConfig.mk
+++ b/groups/kernel/BoardConfig.mk
@@ -5,6 +5,8 @@ else ifeq ($(BASE_LINUX_INTEL_LTS2023_KERNEL), true)
   TARGET_BOARD_KERNEL_HEADERS := $(INTEL_PATH_COMMON)/{{{linux_intel_lts2023_src_path}}}/kernel-headers
 else ifeq ($(BASE_LTS2024_ANDROID_KERNEL), true)
   TARGET_BOARD_KERNEL_HEADERS := $(INTEL_PATH_COMMON)/{{{lts2024_android_src_path}}}/kernel-headers
+else ifeq ($(BASE_LINUX_INTEL_LTS2024_KERNEL), true)
+  TARGET_BOARD_KERNEL_HEADERS := $(INTEL_PATH_COMMON)/{{{linux_intel_lts2024_src_path}}}/kernel-headers
 else
   TARGET_BOARD_KERNEL_HEADERS := $(INTEL_PATH_COMMON)/{{{src_path}}}/kernel-headers
 endif
diff --git a/groups/kernel/option.spec-gmin b/groups/kernel/option.spec-gmin
index 8d803638..d8d0fda7 100644
--- a/groups/kernel/option.spec-gmin
+++ b/groups/kernel/option.spec-gmin
@@ -32,6 +32,8 @@ linux_intel_lts2023_src_path =
 linux_intel_lts2023_cfg_path =
 lts2024_android_src_path =
 lts2024_android_cfg_path =
+linux_intel_lts2024_src_path =
+linux_intel_lts2024_cfg_path =
 more_modules = false
 branch =
 kmi_generation =
-- 
2.34.1

