From 0ff550774c1ea2fc01f0dc59aaa1ca4cc18bc6f3 Mon Sep 17 00:00:00 2001
From: "Kalenahally Madegowda, NandishaX"
 <nandishax.kalenahally.madegowda@intel.com>
Date: Thu, 3 Mar 2022 16:25:03 +0530
Subject: [PATCH] Enable GUC for HWC support

Signed-off-by: Kalenahally Madegowda, NandishaX <nandishax.kalenahally.madegowda@intel.com>

diff --git a/groups/graphics/auto/BoardConfig.mk b/groups/graphics/auto/BoardConfig.mk
index f3d6c15..60da9e1 100755
--- a/groups/graphics/auto/BoardConfig.mk
+++ b/groups/graphics/auto/BoardConfig.mk
@@ -5,7 +5,11 @@ LIBDRM_VER ?= intel
 BOARD_KERNEL_CMDLINE += vga=current i915.modeset=1 drm.atomic=1 i915.nuclear_pageflip=1 drm.vblankoffdelay=1 i915.fastboot=1
 {{^acrn-guest}}
 {{#enable_guc}}
+ifeq ($(BASE_LTS2020_YOCTO_KERNEL),true)
+BOARD_KERNEL_CMDLINE += i915.enable_guc=1
+else
 BOARD_KERNEL_CMDLINE += i915.enable_guc=2
+endif
 {{/enable_guc}}
 {{/acrn-guest}}

@@ -13,10 +17,6 @@ ifeq ($(BASE_YOCTO_KERNEL),true)
 BOARD_KERNEL_CMDLINE += i915.enable_guc=2
 endif

-ifeq ($(BASE_LTS2020_YOCTO_KERNEL),true)
-BOARD_KERNEL_CMDLINE += i915.enable_guc=1
-endif
-
 USE_OPENGL_RENDERER := true
 USE_INTEL_UFO_DRIVER := false
 BOARD_GPU_DRIVERS := i965 swrast virgl iris
-- 
2.35.1

