From 56b277d5ce158f27b9aad3842727de690ea11a29 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Thu, 10 Feb 2022 13:17:36 +0530
Subject: [PATCH] Revert "Enable keymaster 3.0"

This reverts commit d345d7f0d41f8bc5da682719aecc4986aa2bcc9f.

diff --git a/groups/trusty/false/product.mk b/groups/trusty/false/product.mk
index d1b8cbc..e112165 100644
--- a/groups/trusty/false/product.mk
+++ b/groups/trusty/false/product.mk
@@ -1,3 +1,2 @@
 PRODUCT_PACKAGES += \
-    android.hardware.gatekeeper@1.0-service.software \
-    android.hardware.keymaster@3.0-service
+    android.hardware.gatekeeper@1.0-service.software
diff --git a/groups/trusty/true/option.spec b/groups/trusty/true/option.spec
index b6cc701..f8b0f50 100644
--- a/groups/trusty/true/option.spec
+++ b/groups/trusty/true/option.spec
@@ -2,4 +2,5 @@
 lk_project       = sand-x86-64
 enable_hw_sec    = true
 enable_storage_proxyd = true
+keymaster_version   = 2
 ref_target =
diff --git a/groups/trusty/true/product.mk b/groups/trusty/true/product.mk
index 18cdcfd..c2acdf2 100644
--- a/groups/trusty/true/product.mk
+++ b/groups/trusty/true/product.mk
@@ -1,12 +1,25 @@
 {{#enable_hw_sec}}
 
+KM_VERSION := {{{keymaster_version}}}
+
+ifeq ($(KM_VERSION),2)
+PRODUCT_PACKAGES += \
+	keystore.trusty
+PRODUCT_PROPERTY_OVERRIDES += \
+	ro.hardware.keystore=trusty
+endif
+
+ifeq ($(KM_VERSION),1)
+PRODUCT_PACKAGES += \
+	keystore.${TARGET_BOARD_PLATFORM}
+endif
+
 PRODUCT_PACKAGES += \
 	libtrusty \
 	storageproxyd \
 	libinteltrustystorage \
 	libinteltrustystorageinterface \
 	android.hardware.gatekeeper@1.0-service.trusty \
-	android.hardware.keymaster@3.0-service.trusty \
 	keybox_provisioning \
 
 PRODUCT_PACKAGES_DEBUG += \
@@ -17,5 +30,4 @@ PRODUCT_PACKAGES_DEBUG += \
 
 PRODUCT_PROPERTY_OVERRIDES += \
 	ro.hardware.gatekeeper=trusty \
-	ro.hardware.keystore=trusty
 {{/enable_hw_sec}}
-- 
2.35.1

