From f3fb7617972e8e35014c154ee0307aecfe1cd062 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Sun, 13 Nov 2022 21:11:09 +0530
Subject: [PATCH] [ADL]Fix for GTS checkVulkanDeqpLevelIsHighEnough

Tracked-On: OAM-104706

diff --git a/groups/graphics/auto/product.mk b/groups/graphics/auto/product.mk
index e17d8dc..0f893a2 100755
--- a/groups/graphics/auto/product.mk
+++ b/groups/graphics/auto/product.mk
@@ -94,7 +94,7 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.opengles.aep.xml:vendor/etc/permissions/android.hardware.opengles.aep.xml
 
 PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.opengles.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.opengles.deqp.level.xml
+    frameworks/native/data/etc/android.software.opengles.deqp.level-2021-03-01.xml:vendor/etc/permissions/android.software.opengles.deqp.level.xml
 
 # GLES version
 PRODUCT_PROPERTY_OVERRIDES += \
@@ -115,7 +115,7 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.vulkan.version-1_1.xml:vendor/etc/permissions/android.hardware.vulkan.version.xml
 
 PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.vulkan.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
+    frameworks/native/data/etc/android.software.vulkan.deqp.level-2021-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
 
 PRODUCT_PACKAGES += \
     vulkan.$(TARGET_BOARD_PLATFORM) \
-- 
2.38.1

