From 812a5040bfbe282efa70b1c5a5191b5b3d307199 Mon Sep 17 00:00:00 2001
From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
Date: Thu, 17 Nov 2022 15:37:19 +0530
Subject: [PATCH] Fix logic for enabling VFs in vm-manager

Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>
---
 src/guest/start.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/guest/start.c b/src/guest/start.c
index 05f5907..c64fced 100644
--- a/src/guest/start.c
+++ b/src/guest/start.c
@@ -428,7 +428,7 @@ static int set_available_vf(void)
         close(fd);
         numvfs = strtoul(buf, NULL, 10);
 
-	if (numvfs < total_vfs) {
+	if (numvfs == 0) {
 		memset(buf, 0, sizeof(buf));
 		snprintf(buf, sizeof(buf), "%d", total_vfs);
 
-- 
2.38.1

