From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Mon, 14 Mar 2022 15:44:15 +0800
Subject: [PATCH] Mixins update - Enable GUC for HWC support

---

diff --git a/caas/BoardConfig.mk b/caas/BoardConfig.mk
index 41bd90e..877134b 100644
--- a/caas/BoardConfig.mk
+++ b/caas/BoardConfig.mk
@@ -485,13 +485,14 @@ TARGET_USE_PRIVATE_LIBDRM := true
 LIBDRM_VER ?= intel

 BOARD_KERNEL_CMDLINE += vga=current i915.modeset=1 drm.atomic=1 i915.nuclear_pageflip=1 drm.vblankoffdelay=1 i915.fastboot=1
-
-ifeq ($(BASE_YOCTO_KERNEL),true)
+ifeq ($(BASE_LTS2020_YOCTO_KERNEL),true)
+BOARD_KERNEL_CMDLINE += i915.enable_guc=1
+else
 BOARD_KERNEL_CMDLINE += i915.enable_guc=2
 endif

-ifeq ($(BASE_LTS2020_YOCTO_KERNEL),true)
-BOARD_KERNEL_CMDLINE += i915.enable_guc=1
+ifeq ($(BASE_YOCTO_KERNEL),true)
+BOARD_KERNEL_CMDLINE += i915.enable_guc=2
 endif

 USE_OPENGL_RENDERER := true
-- 
2.17.1

