From 47e8b862fbf89a2325026c13133b5d3711cf2573 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Sun, 13 Nov 2022 21:07:58 +0530
Subject: [PATCH] Mixin up for GTS checkVulkanDeqpLevelIsHighEnough

Tracked-On: OAM-104706

diff --git a/caas/device.mk b/caas/device.mk
index 6e2729c..4337e22 100644
--- a/caas/device.mk
+++ b/caas/device.mk
@@ -474,7 +474,7 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.opengles.aep.xml:vendor/etc/permissions/android.hardware.opengles.aep.xml
 
 PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.opengles.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.opengles.deqp.level.xml
+    frameworks/native/data/etc/android.software.opengles.deqp.level-2021-03-01.xml:vendor/etc/permissions/android.software.opengles.deqp.level.xml
 
 # GLES version
 PRODUCT_PROPERTY_OVERRIDES += \
@@ -491,7 +491,7 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.vulkan.version-1_1.xml:vendor/etc/permissions/android.hardware.vulkan.version.xml
 
 PRODUCT_COPY_FILES += \
-    frameworks/native/data/etc/android.software.vulkan.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
+    frameworks/native/data/etc/android.software.vulkan.deqp.level-2021-03-01.xml:vendor/etc/permissions/android.software.vulkan.deqp.level.xml
 
 PRODUCT_PACKAGES += \
     vulkan.$(TARGET_BOARD_PLATFORM) \
-- 
2.38.1

